src/test/java/jp/ac/utokyo/rcast/karkinos/alleliccnv/WaveletDenoizeACNVTest.java

```diff
@@ -0,0 +1,398 @@
+                                0.5f, 0.8f, 0.6f, 0.9f, 0.8f, 0.6f,
+                                0.7f, 0.5f, 0.3f, 0.9f, 0.8f, 0.6f}},
+                        new float[][][] {{{1.0088658f, 0.9338659f},
+                                {1.0088658f, 0.9338659f}, {1.0088658f, 0.9338659f},
+                                {1.0088658f, 0.9338659f}, {1.0088658f, 0.9338659f},
+                                {1.0f, 0.9250001f}, {1.0f, 0.9875f}, {1.0f, 0.9875f},
+                                {1.0213659f, 0.9088659f}, {1.0088658f, 0.8713659f},
+                                {0.9963659f, 0.9963659f}, {1f, 0.9875001f},
+                                {1f, 0.9875001f}, {1.0213659f, 0.9838659f},
+                                {1.0463659f, 0.9463659f}, {1.0f, 0.975f}, {1f, 0.9875f},
+                                {1.0f, 0.92499995f}, {1.0588659f, 0.9088659f},
+                                {1.0713658f, 0.9338659f}, {1.0713658f, 0.8963659f},
+                                {1.0713658f, 0.8963659f}, {1.0713658f, 0.8963659f},
+                                {1.0713658f, 0.8963659f}}}),
+                Arguments.of(
+                        new float[][] {{0.1f, 0.1f, 0.1f, 0.1f, 0.1f, 0.1f,
+                                0.1f, 0.1f, 0.1f, 0.1f, 0.1f, 0.1f,
+                                0.1f, 0.1f, 0.1f, 0.1f, 0.1f, 0.1f,
+                                0.1f, 0.1f, 0.1f, 0.1f, 0.1f, 0.1f}},
+                        new float[][] {{1.4f, 1.5f, 1.3f, 1.1f, 1.4f, 1.6f,
+                                1.2f, 1.4f, 0.9f, 1.4f, 1.5f, 1.6f,
+                                1.3f, 0.7f, 1.4f, 1.6f, 1.5f, 1.6f,
+                                0.8f, 1.4f, 1.6f, 1.4f, 1.5f, 1.6f}},
+                        new float[][] {{1.4f, 1.5f, 1.3f, 1.1f, 1.4f, 1.6f,
+                                1.2f, 1.4f, 0.9f, 1.4f, 1.5f, 1.6f,
+                                1.3f, 0.7f, 1.4f, 1.6f, 1.5f, 1.6f,
+                                0.8f, 1.4f, 1.6f, 1.4f, 1.5f, 1.6f}},
+                        new float[][][] {{{1f, 1f}, {1f, 1f}, {1f, 1f}, {1f, 1f},
+                                {1f, 1f}, {1f, 1f}, {1f, 1f}, {1f, 1f}, {1f, 1f},
+                                {1f, 1f}, {0.9963659f, 0.9963659f}, {1f, 1f},
+                                {1f, 1f}, {1f, 1f}, {1f, 1f}, {1f, 1f}, {1f, 1f},
+                                {1f, 1f}, {1f, 1f}, {1f, 1f}, {1f, 1f}, {1f, 1f},
+                                {1f, 1f}, {1f, 1f}}}));
+    }
+
+    @ParameterizedTest
+    @MethodSource("calcParameters")
+    void calcTest(final float[][] rows,
+                  final float[][] gcAdjustedHs,
+                  final float[][] gcAdjustedLs,
+                  final float[][][] expWtValHLs) throws IOException {
+        List<List<SNVHolderPlusACnv>> pList = IntStream.range(0, rows.length).mapToObj(i ->
+                IntStream.range(0, rows[i].length).mapToObj(j -> {
+                    ACNVInfoBean highEra = new ACNVInfoBean();
```

Comment:
I guess this should be 'higherA': higher allele.

Reply:
Thank you pointed out.
Fixed.